Fix warning: incompatible implicit declaration of
built-in function 'free'
By OKUMURA N. Shin-ya, oku.ns@dream.com
https://bugs.gentoo.org/show_bug.cgi?id=264884
Upstream status unknown.


Index: texlive-20080816-source/texk/web2c/xetexdir/jpegimage.c
===================================================================
--- texlive-20080816-source.orig/texk/web2c/xetexdir/jpegimage.c
+++ texlive-20080816-source/texk/web2c/xetexdir/jpegimage.c
@@ -80,6 +80,7 @@
 #include "jpegimage.h"
 
 #include <memory.h>
+#include <stdlib.h>
 
 #include "kpathsea/lib.h" /* for xmalloc/xrealloc prototypes */
 
